src/api/db/migrate/20170630144825_convert_tokens_to_service_tokens.rb

```diff
@@ -0,0 +1,9 @@
+class ConvertTokensToServiceTokens < ActiveRecord::Migration[5.1]
+  def up
+    Token.where(type: nil).update_all(type: 'Token::Service')
+  end
+
+  def down
+    Token::Service.update_all(type: nil)
```

Comment:
is down really necessary? As when we revert this migration we drop the field anyway?
But it is fine for me to leave it, just as a comment!